rskj-core/src/main/java/co/rsk/core/bc/BlockChainImpl.java

```diff
@@ -156,7 +157,15 @@ public ImportResult tryToConnect(Block block) {
                     long saveTime = System.nanoTime();
                     ImportResult result = internalTryToConnect(block);
                     long totalTime = System.nanoTime() - saveTime;
-                    logger.info("block: num: [{}] hash: [{}], processed after: [{}]nano, result {}", block.getNumber(), block.getPrintableHash(), totalTime, result);
+                    String timeInSeconds = FormatUtils.formatNanosecondsToSeconds(totalTime);
+
+                    if (BlockUtils.tooMuchProcessTime(totalTime)) {
+                        logger.warn("block: num: [{}] hash: [{}], processed after: [{}]seconds, result {}", block.getNumber(), block.getPrintableHash(), timeInSeconds, result);
```

Comment:
you don't need to add num and hash, MDC adds that to every logged line

Reply:
OK, but why such messages were not changed in the pull request that adds the MDC? The above messages are only an improve over those messages